packages/abi-typegen/src/utils/transpile-abi.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Duplicate files like this are all the more reason for https://github.com/FuelLabs/fuels-ts/issues/2346 which I think should be a part of the checklist in https://github.com/FuelLabs/fuels-ts/issues/2493

Reply:
Noted in [here](https://github.com/FuelLabs/fuels-ts/issues/2493#issuecomment-2269946371)